boards/nxp/imx93_evk/imx93_evk_mimx9352_a55_defconfig

```diff
@@ -29,3 +29,6 @@ CONFIG_UART_CONSOLE=y
 
 CONFIG_CLOCK_CONTROL=y
 CONFIG_PINCTRL=y
+
+# CAN Phy must be enabled after mfd0 ready
+CONFIG_CAN_TRANSCEIVER_INIT_PRIORITY=75
```

Comment:
As `CONFIG_CAN_TRANSCEIVER_INIT_PRIORITY` depends on `CONFIG_CAN`, you should move this change of default value to `boards/nxp/imx93_evk/Kconfig.defconfig` as a conditional.

Reply:
Could you please be more specific?

I tried to create `boards/nxp/imx93_evk/Kconfig.defconfig` with following content:

```c
if BOARD_IMX93_EVK_MIMX9352_A55

if CAN
# CAN Phy must be enabled after mfd0 gpio expander is ready
config CAN_TRANSCEIVER_INIT_PRIORITY
    default 75
endif

endif # BOARD_IMX93_EVK_MIMX9352_A55
```

and check the `.config` file under build directory, only to find `CONFIG_CAN_TRANSCEIVER_INIT_PRIORITY` still default to 45.

Not really that familiar with Kconfig, did I miss something?

Reply:
> and check the `.config` file under build directory, only to find `CONFIG_CAN_TRANSCEIVER_INIT_PRIORITY` still default to 45.

Did you do a completely clean build? You are only chasing the default, so rebuilding after this change will have no effect.

Reply:
after deleting build directory, still no effect.

Reply:
Since the file in question is not included in this PR, I can't really try reproducing the issue, you are seeing.

Reply:
I'll push my revision, so that you could check that.

Reply:
The changes pushed so far - are they not working?

Reply:
no, it's not.

Reply:
my bad, it is actually working...